soc/ti/k3/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
 # Copyright (c) 2024 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
+zephyr_include_directories(common)
```

Comment:
This can mess up things, better to be clear and use the full path in the code below,

```
#include <common/k3_mmr.h>
```